.changeset/wardley-ecosystem-and-attitudes.md

```
@@ -0,0 +1,24 @@
+Extends the `wardley-beta` diagram to match more of the OWM syntax:
+
+- `(ecosystem)` decorator on a component, drawn as concentric circles with a diagonal-hatch ring (matches OWM's ecosystem-play symbol).
+- Attitude zones using OWM's canonical 4-coordinate form `keyword [v1, m1, v2, m2]` for the two opposing corners. Renders as a translucent labelled rectangle in the OWM colour palette:
+  - `pioneers` — light blue
+  - `settlers` — medium blue
+  - `townplanners` — purple
+- `explorers` is accepted as an alias for `pioneers`, `villagers` as an alias for `settlers`.
+
+Example:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)